config.json

```diff
@@ -517,6 +523,12 @@
             "ContainerName": "default"
           }
         },
+        "nodeReuse":{
```

Comment:
Yes I think that would be a good way to both specify default commands for tools as well as to make them unique for the different platforms.